### ISSUE TITLE
How to delete an anchor created with  mSession.createAnchor() ?

### ISSUE THREAD

**Issue by RegisAG** [google-ar/arcore-android-sdk#246]
Is it possible to remove an anchor created with  mSession.createAnchor()  ?

So that it does not appear anymore in getAllTrackables.

Thanks !

**Comment by inio** [comment]
I believe [Anchor.detach()](https://developers.google.com/ar/reference/java/com/google/ar/core/Anchor.html\#detach\(\)) does this.  Does that work for you?
